scripts/cli/util/schema_test.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 ( set -x; node ./util/schema_check.js args...; ) 2>&1 | tee output.log
 
-errLen=$(cat output.log | grep Unknown | wc -l)
+errLen=$(cat output.log | grep 'Unknown\|SyntaxError' | wc -l)
```

Comment:
Doesn't polkadot.js return an error when there's SyntaxError that you can catch instead of greping the o/p? There might be other errors as well that we should catch.